invoice/src/main/java/org/killbill/billing/invoice/config/MultiTenantInvoiceConfig.java

```diff
@@ -231,6 +230,24 @@ public UsageDetailMode getItemResultBehaviorMode(final InternalTenantContext ten
         return UsageDetailMode.AGGREGATE;
     }
 
+    @Override
+    public InArrearMode getInArrearMode() {
```

Comment:
We'll need to update the table at https://docs.killbill.io/latest/userguide_configuration.html#configuration_properties_table.

Reply:
@pierre See https://github.com/killbill/killbill-docs/pull/459